client/my-sites/comments/comment-list/index.jsx

```diff
@@ -181,13 +181,29 @@ export class CommentList extends Component {
 		this.props.createNotice( type, message, options );
 	}
 
-	submitComment = comment => {
-		this.props.removeNotice( 'comment-submitted' );
-		this.props.createNotice( 'is-success', this.props.translate( 'Comment submitted.' ), {
-			duration: 5000,
-			id: 'comment-submitted',
-			isPersistent: true,
-		} );
+	submitComment = ( comment, options = { alsoApprove: false } ) => {
+		const { translate } = this.props;
+		const { alsoApprove } = options;
+
+		this.props.removeNotice( `comment-notice-${ comment.parentId }` );
+
+		const noticeMessage = alsoApprove
+			? translate( 'Comment approved and reply submitted.' )
+			: translate( 'Reply submitted.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **39** times:
`translate( 'Submitted!' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).